modules/dailymotionBidAdapter.js

```diff
@@ -159,11 +159,17 @@ export const spec = {
         // Vendor consent
         deepAccess(bidderRequest, 'gdprConsent.vendorData.vendor.consents.573') === true &&
         // Purposes
-        [1, 3, 4].every(v => deepAccess(bidderRequest, `gdprConsent.vendorData.purpose.consents.${v}`) === true) &&
+        [1, 3, 4].every(v =>
+          deepAccess(bidderRequest, `gdprConsent.vendorData.publisher.consents.${v}`) !== false &&
```

Comment:
Are we sure this value is always filled with a `false` ? 

Just to make sure, `!== false` will return `true` in many cases, such as `undefined !== false`, `'false' !== false`, `null !== false`, etc. 

So if you want to verify if this value exists AND is false (or not true) maybe we should add an other condition.

Reply:
Hello, this is the expected behaviour : these entries are constraints that can disregard the consent only if present AND `false`.
If the value is not there (or present but `true`), we evaluate the other consent values (these ones must be `===true`)

Reply:
Ok perfect, thank you 👍🏻